README.md

```diff
@@ -1322,42 +1322,44 @@ For more, see [pygrep-hooks](https://github.com/pre-commit/pygrep-hooks) on GitH
+| PLE0604 | [invalid-all-object](https://pylint.readthedocs.io/en/latest/user_guide/messages/error/invalid-all-object.html) | Invalid object in `__all__`, must contain only strings |  |
+| PLE0605 | [invalid-all-format](https://pylint.readthedocs.io/en/latest/user_guide/messages/error/invalid-all-format.html) | Invalid format for `__all__`, must be `tuple` or `list` |  |
+| PLE1142 | [await-outside-async](https://pylint.readthedocs.io/en/latest/user_guide/messages/error/await-outside-async.html) | `await` should be used within an async function |  |
 
 #### Refactor (PLR)
 
 | Code | Name | Message | Fix |
 | ---- | ---- | ------- | --- |
-| PLR0133 | constant-comparison | Two constants compared in a comparison, consider replacing `{left_constant} {op} {right_constant}` |  |
-| PLR0206 | property-with-parameters | Cannot have defined parameters for properties |  |
-| PLR0402 | consider-using-from-import | Use `from {module} import {name}` in lieu of alias | 🛠 |
-| PLR0913 | too-many-args | Too many arguments to function call ({c_args}/{max_args}) |  |
-| PLR0915 | too-many-statements | Too many statements ({statements}/{max_statements}) |  |
-| PLR1701 | consider-merging-isinstance | Merge these isinstance calls: `isinstance({obj}, ({types}))` |  |
-| PLR1722 | use-sys-exit | Use `sys.exit()` instead of `{name}` | 🛠 |
-| PLR2004 | magic-value-comparison | Magic value used in comparison, consider replacing {value} with a constant variable |  |
+| PLR0133 | [constant-comparison](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/constant-comparison.html) | Two constants compared in a comparison, consider replacing `{left_constant} {op} {right_constant}` |  |
+| PLR0206 | [property-with-parameters](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/property-with-parameters.html) | Cannot have defined parameters for properties |  |
+| PLR0402 | [consider-using-from-import](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/consider-using-from-import.html) | Use `from {module} import {name}` in lieu of alias | 🛠 |
+| PLR0913 | [too-many-args](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/too-many-args.html) | Too many arguments to function call ({c_args}/{max_args}) |  |
+| PLR0915 | [too-many-statements](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/too-many-statements.html) | Too many statements ({statements}/{max_statements}) |  |
+| PLR1701 | [consider-merging-isinstance](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/consider-merging-isinstance.html) | Merge these isinstance calls: `isinstance({obj}, ({types}))` |  |
+| PLR1722 | [use-sys-exit](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/use-sys-exit.html) | Use `sys.exit()` instead of `{name}` | 🛠 |
```

Comment:
The real name seems to be ``consider-using-sys-exit``, there's other broken links, I'll fix them if the PR is deemed valuable. (Otherwise I can fix in another MR)

Reply:
Okay cool -- we can safely rename these as they aren't part of the public API in any way yet.